.github/workflows/autotest.yml

```diff
@@ -19,6 +19,52 @@ jobs:
         shell: bash
         run: |
           python3 tools/add_header --dry-run --disable-progress-bar
+  python_annotations:
+    name: Check Python Annotations
+    runs-on: ubuntu-latest
+    env:
+      UPDATED_PYTHON_ANNOTATIONS_REPORT_FILE: ../updated_annotation_report.txt
+      INITIAL_PYTHON_ANNOTATIONS_REPORT_FILE: ../initial_annotation_report.txt
+    steps:
+      - uses: actions/checkout@v2
+      - name: Count current number of missing annotations
+        id: count_updated
+        shell: bash
+        run: |
+          pip install flake8 flake8-annotations
```

Comment:
I would move this to `tools/count_missing_annotations.py`. This way if a dev runs the script locally, it would not fail ;)

Reply:
Okay, I changed the behavior a bit:

The `count_missing_annotations.py` script proceeds normally if all required packages are installed.
If there is a package missing, it installs it temporarily. The package will be removed once the scripts ends (e.g. error, normal exit, abort, ...). This way, users can run the script without enabling this feature of flake8 forever.

I would leave the installation of `flake8-annotations` at this point, so it can be used as a "cached" version and does not have to be installed every time the script runs.